### ISSUE TITLE
Makefile dictionary

### ISSUE THREAD

**Issue by andrewvaughan** [streetsidesoftware/cspell-dicts#2665]
Makefiles currently do not have support (as far as I can tell)?

I'm confused, because the VSCode extension lists it as an option, but I can't find it anywhere for the life of me. I have an issue in both repositories, as it would be helpful to have a dictionary published, but the fact that it is "supported" in VSCode seems like a bug?

Here it is for reference:
https://github.com/streetsidesoftware/vscode-spell-checker/issues/2883

**Comment by Jason3S** [comment]
@andrewvaughan,

Thank you for raising the issue. The extension allows you to enable/disable spell checking various file types. I believe `makefile` is one of them in the default list.

There is not currently a keyword list / dictionary for makefiles. A PR with keywords and common terms would be most welcome. Are you referring to [GNU make](https://www.gnu.org/software/make/manual/html_node/Quick-Reference.html)?

If you are open to helping, please see [How to create a new Dictionary](https://github.com/streetsidesoftware/cspell-dicts#how-to-create-a-new-dictionary)

**Comment by andrewvaughan** [comment]
Thanks - I guess I was confused - given the UI for the VSCode extension seemingly has a way to add `makefile` support, I figured you were pulling in a `makefile` dictionary from somewhere. If the intent of the UI is to purely enable file extensions, I didn't find that very clear.

Unfortunately, I don't have the bandwidth to create a full `Makefile` dictionary, at the moment. I'll leave this open if someone can pick it up. I've created a starter here, but it is extremely limited to the tokens my project is currently using:

https://github.com/andrewvaughan/template-core/blob/main/.config/dictionaries/ops-deps/makefile.txt
